src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const pageNumbers = [];
+
+  for (let i = 1; i <= Math.ceil(total / perPage); i += 1) {
```

Comment:
Consider replacing the for loop with a more functional approach using `Array.from` or `Array.prototype.map`.